description.txt

```diff
@@ -1 +1,3 @@
-This is a description file for git training session, now resolve conflicts 
\ No newline at end of file
+This is a description file for git training session, now resolve conflicts 
+
+Learning and making Changes in Description file.
```

Comment:
@aakritigkmit You have to add a description of the things you have learnt in yesterday's session.